app/Rules/DockerImageFormat.php

```
@@ -0,0 +1,41 @@
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+
+class DockerImageFormat implements ValidationRule
+{
+    /**
+     * Run the validation rule.
+     *
+     * @param  \Closure(string, ?string=): \Illuminate\Translation\PotentiallyTranslatedString  $fail
+     */
+    public function validate(string $attribute, mixed $value, Closure $fail): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)